data/web/autodiscover.php

```diff
@@ -171,6 +171,16 @@
         <SSL>on</SSL>
         <AuthRequired>on</AuthRequired>
       </Protocol>
+      <Protocol>
+        <Type>POP3</Type>
+        <Server><?=$autodiscover_config['pop3']['server'];?></Server>
+        <Port><?=$autodiscover_config['pop3']['port'];?></Port>
+        <DomainRequired>off</DomainRequired>
+        <LoginName><?=$email;?></LoginName>
+        <SPA>off</SPA>
+        <SSL>on</SSL>
+        <AuthRequired>on</AuthRequired>
+      </Protocol>
```

Comment:
This is inside the `$autodiscover_config['autodiscoverType'] == 'imap'` section, which I don't think makes sense. Does Outlook even support having both IMAP and POP3 in the same discovery? Which one does it pick?